Language_Parser/itemDictionary.json

```diff
@@ -37,14 +37,16 @@
     "mouse": ["mouse"],
     "movies": ["movies", "movie"],
     "noise": ["noise", "tapping"],
-    "polaroid2": ["oldpolaroid"],
     "painting": ["painting", "portrait", "paintings", "portraits", "frame", "frames", "firstpainting", "secondpainting", "thirdpainting"],
     "pepper": ["pepper"],
     "pillow": ["pillow", "pillows"],
     "pills": ["pills", "pill", "pillbottle", "bottle", "medicine", "medicinebottle", "medication"],
     "plates": ["plates", "paperplates"],
     "polaroid": ["polaroid", "polaroids", "picture"],
+    "polaroid2": ["oldpolaroid"],
+    "polaroid3": ["dustypolaroid"],
     "rack": ["rack", "coatrack"],
```

Comment:
Thank you for moving these. I totally forgot to do that when I changed them from old/dusty polaroid to polaroid2/3

Reply:
I got you!